cmd/influx/config/mock.go

```diff
@@ -2,8 +2,9 @@ package config
 
 // MockConfigService mocks the ConfigService.
 type MockConfigService struct {
```

Comment:
why is this exported?

Reply:
for cli level testing

Reply:
then move the mock to the CLI, doesn't belong at the produce side, belongs at the consumer side generally